web-animations-1/Overview.bs

```diff
@@ -620,6 +620,9 @@ purpose of synchronization.
 
 At any given moment, a [=timeline=] has a single current [=time value=] known
 simply as the timeline's <dfn lt="timeline current time">current time</dfn>.
+The <dfn lt="timeline duration">duration</a> of a timeline gives the
+maximum value a timeline may generate for
+[=timeline current time|current time=].
```

Comment:
Perhaps we could make this a new paragraph and explain a little what it's purpose is?